core/include/core/G3Timestream.h

```diff
@@ -5,7 +5,7 @@
 #include <G3TimeStamp.h>
 #include <G3Vector.h>
 #include <vector>
-#include <map>
+#include <containers.h>
```

Comment:
Is there a way to avoid including containers.h here? It might be nice to hide this as an internal detail. I'm not sure it is possible though.